fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1902,8 +1902,9 @@ public void testIngestOnSubtree() throws IOException {
      */
     @Test
     public void testIngestOnPairtree() throws IOException {
-        //  Following the approach undertaken for FedoraExportIT#shouldRoundTripOnePairtree
-        final String objName = getLocation(postObjMethod());
+        final String parent = "123456/789/10";
+        executeAndClose(putObjMethod(parent));
```

Comment:
This test is no longer valid with these changes.

It seems like if the test is still required then we should create a separate test class which runs with the old pidminter setup so we could still test pairtrees.

Not sure, @awoods

Reply:
Yeah, I figured just telling the PID to have a `/` in it wouldn't create a pairtree, but I'm not really sure. Hence not blocking this. I'm fine, if this test is still useful.